src/components/dataset/overview/DatasetOverviewPanel.jsx

```diff
@@ -146,16 +146,6 @@ function DatasetOverviewPanel(props) {
             {...a11yProps(3)}
           />
         )}
-        {canViewJournalEntries && (
```

Comment:
Not sure if this was a bad delete, but I think this double was the result of a bad merge?

Reply:
Nice catch! I had noticed this in dev and meant to file a ticket, thanks for fixing.  A bad merge seems plausible.

Reply:
Ah yes! Pulled from develop ✨